### ISSUE TITLE
Any way to save trained process so we don't have to train from the beginning.

### ISSUE THREAD

**Issue by Muhammad-Zubair** [XavierXiao/Dreambooth-Stable-Diffusion#42]
I want to save and load the model from google drive or other resources online can you please add 2 cells for that functionality.
Thanks

**Comment by ClashSAN** [comment]
This repo doesn't maintain a colab. 
You mean to post here: https://github.com/ShivamShrirao/diffusers/blob/main/examples/dreambooth/
and currently converting to ckpt files aren't possible.

**Comment by Fikxzer** [comment]
Yes, this repository creates ckpt files. I just point my new training to the ckpt that was created in the logs folder from previous training instead of the original full ema model. It will resume training on the model you pick and then create a new model in the log folder. I even copy these ckpt files to a different stable diffusion webui build and use the trained model with web UI.

**Comment by Fikxzer** [comment]
ckpt files are stored in logs folder checkpoints for this repo.
